internal/myks/assets/data-schema.ytt.yaml

```
@@ -121,3 +121,11 @@ myks:
   gitRepoBranch: ""
   #! Set to the current git repository URL if available.
   gitRepoUrl: ""
+#! meta data of the current rendering step
+meta:
+  #! application name currently being rendered
+  app: ""
+  #! name of prototype including full path (e.g. prototypes/argocd)
+  prototype: ""
+  #! rendering step: argocd, global_ytt, helm, ytt-pkg, ytt
+  step: ""
```

How about this? I haven't thought thoroughly about it but it feels like the idea can be used also in the sync step. I also moved it into the `myks` section of the schema to show that it's something users shouldn't touch.

```suggestion
  #! context of the current operation
  context:
    #! application name currently being processed
    app: ""
    #! name of prototype including full path (e.g. prototypes/argocd)
    prototype: ""
    #! rendering step: argocd, global_ytt, helm, ytt-pkg, ytt
    step: ""

```